layouts/product/pulumi-insights.html

```diff
@@ -64,7 +97,7 @@ <h4 class="p-0 m-0 text-gray-800">{{ .subtitle }}</h4>
         </div>
     </section>
 
-    <section id="analytics" class="container my-28 mx-auto text-center px-6">
+    <section id="analytics" class="container my-20 mx-auto text-center px-6">
```

Comment:
I don't think we need this section. I think we can delete it. I do like the shadowed box it is using. Can we drop the pricing section into a similar box?